ember-accounts/app/templates/components/x-account.hbs

```diff
@@ -0,0 +1,89 @@
+{{#if showModal}}
+ {{#x-modal}}
+    {{component currentModal model=modalModel close='closeModal'}}
+  {{/x-modal}}
+{{/if}}
+
+<div class="alert {{if errorState 'alert-error' 'alert-success'}}">
+  <div class={{if trialing 'trialing' 'disabled'}}>
+    <p><strong> Free Trial is Active! </strong></p>
+    <p>Expires: {{stripe-date plan.trial_end}}</p>
+  </div>
+  <div class={{if trialingExpired 'trialing-expired' 'disabled'}}>
+    <p><strong> Free Trial is Expired! </strong></p>
+    <p> Upgrade your plan to enable private access </p>
+  </div>
+
+
+  <div class={{if noAccount 'new-account' 'disabled'}}>
+    <div>
+      <p>
+        <strong>All Accounts</strong> receive a <strong>15 day free trial</strong>.
+      </p>
+    </div>
+    <div class='trial-form'>
+      <form {{action "activateTrial" on="submit"}}>
+        {{input type="email" placeholder="Email Address" name="billing_email" value=emailBinding required="required"}}
```

Comment:
Rationale is that one click is easier than type + click. I would guess for most orgs the billing contact for GitHub is also the one that would be responsible for paying for HuBoard.

That said, this is the kind of thing we could A/B test too. Leave it for now.